### PR TITLE
Handle missing backups

### PR DESCRIPTION
**Pull request by moelsayed**
https://github.com/rancher/rancher/issues/18419

### CODE REVIEW

File: cluster/etcd.go [hunk 1]
```diff
@@ -41,11 +42,21 @@ func (c *Cluster) PrepareBackup(ctx context.Context, snapshotPath string) error
 			if backupServer == nil { // start the download server, only one node should have it!
 				if err := services.StartBackupServer(ctx, host, c.PrivateRegistriesMap, backupImage, snapshotPath); err != nil {
 					log.Warnf(ctx, "failed to start backup server on host [%s]: %v", host.Address, err)
+					errors = append(errors, err)
 					continue
 				}
 				backupServer = host
```
Comment (galal-hussein) [inline cluster/etcd.go]: Should we break here if the backup server succeed to start

Comment (moelsayed) [reply cluster/etcd.go]: fixed.

File: util/util.go [hunk 2]
```diff
@@ -7,13 +7,13 @@ import (
...
 	"github.com/coreos/go-semver/semver"
-	"github.com/rancher/types/apis/management.cattle.io/v3"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
 )
 
 const (
 	WorkerThreads = 50
 
-	SupportedSyncToolsVersion = "0.1.22"
+	SupportedSyncToolsVersion = "0.1.25"
```
Comment (alena1108) [inline util/util.go]: I feel like we should use min version. Otherwise it's hard to keep up

Comment (moelsayed) [reply util/util.go]: It's an after/before version. We don't need to update this once it's released.

**Pull request merged**